pkg/pluginmanager/manager.go

```diff
@@ -545,48 +545,103 @@ func InitializePlugin(plugin *cli.PluginInfo) error {
 	return nil
 }
 
-// InstallPlugin installs a plugin by name, version and target.
+// InstallStandalonePlugin installs a plugin by name, version and target as a standalone plugin.
+func InstallStandalonePlugin(pluginName, version string, target configtypes.Target) error {
+	return installPlugin(pluginName, version, target, "")
+}
+
+// InstallPluginFromContext installs a plugin by name, version and target as a context-scope plugin.
+func InstallPluginFromContext(pluginName, version string, target configtypes.Target, contextName string) error {
+	if contextName == "" {
+		log.V(4).Warning("Missing context name for a context-scope plugin: %s/%s/%s", pluginName, version, string(target))
```

Comment:
Currently, we do not have a way to select the log level when invoking the command. So, this log should be printed at normal log level without `V(4)`.

Reply:
If we hit this case it probably means we have done a coding error where we called `InstallPluginFromContext()`.
So as Vui says, there is probably nothing a user can do, which is why I didn't want it to show all the time.  But I did want to have some way to help troubleshoot it...

I will remove the `V(4)` since we don't have a way to configure the logging level yet.